tests/v3_api/resource/openldap.json

```diff
@@ -52,5 +52,9 @@
   ],
   "testautogroupnested1": [
     "testauto1"
+  ],
+  "specialcharacter_users": [
+    "testautouser9",
```

Comment:
same as above, no special character?

Reply:
testautouser9 has special characters in `cn` and `sn`